imap_processing/lo/packet_definitions/P_ILO_BOOT_MEMDMP.xml

```diff
@@ -28,7 +28,10 @@
 				<xtce:UnitSet />
 				<xtce:BinaryDataEncoding bitOrder="mostSignificantBitFirst">
 					<xtce:SizeInBits>
-						<xtce:FixedValue>8192</xtce:FixedValue>
+						<xtce:DynamicValue>
```

Comment:
Did you enter these manually? Not a big deal, just wondering in regard to future updates on the generator.

Reply:
Yeah, did all the dynamic value parts manually